api/managementpb/dbaas/xtradb_cluster.proto

```diff
@@ -55,21 +55,13 @@ enum XtraDBBackupState {
 message XtraDBClusterParams {
   // PXC container parameters.
   message PXC {
-    // Requested compute resources.
-    ComputeResources compute_resources = 1 [
-      (validator.field) = {
-        msg_exists: true
-      }
-    ];
+    // Requested compute resources. Do not pass for minikube.
```

Comment:
Well, that's not accurate – minikube _can_ enforce limits, and one _can_ pass them.